src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -96,41 +136,95 @@ def export_config(cmd,
                   dest_connection_string=None,
                   dest_label=None,
                   # to-app-service parameters
-                  appservice_account=None):
+                  appservice_account=None,
+                  skip_features=False):
     # fetch key values from user's configstore
     src_kvs = __read_kv_from_config_store(
         cmd, name=name, connection_string=connection_string, key=key, label=label, prefix_to_remove=prefix)
 
-    # if customer needs preview & confirmation
-    if not yes:
-        # fetch key values from destination
+    # We need to separate KV from feature flags
+    __discard_features_from_retrieved_kv(src_kvs)
+
+    src_features = []
+    dest_features = []
+    dest_kvs = []
+    need_feature_change = False
+
+    if skip_features:
+        if destination == 'appconfig':
+            # dest_kvs contains features and KV that match the label. We discard only features.
+            dest_kvs = __read_kv_from_config_store(
+                cmd, name=dest_name, connection_string=dest_connection_string, key=None, label=dest_label)
```

Comment:
fetch items from destination for comparison under if user requested the preview. If we should check "yes" if it is false then do the rest.